compiler/luci/service/src/Nodes/CircleFullyConnected.cpp

```diff
@@ -36,4 +40,95 @@ luci::CircleNode *CloneNodeLet<CN::DEF>::visit(const luci::CircleFullyConnected
   return cloned;
 }
 
+namespace sinf
+{
+
+loco::TensorShape Algorithm::visit(const luci::CircleFullyConnected *node)
+{
+  auto input_shape = circle_shape(loco::must_cast<CircleNode *>(node->input()));
+  auto weights_shape = circle_shape(loco::must_cast<CircleNode *>(node->weights()));
+
+  loco::TensorShape out_shape;
+
+  // NOTE Some recipes in some repositories are using rank 4 input for FullyConnected.
+  //      Until they are all fixed, disable following assert.
+  // TODO Enable following assert after related fixes are applied
+  // https://github.com/tensorflow/tensorflow/blob/ea33c1e7a25d8025e8ee405ad8ab7be261798d76/tensorflow/lite/kernels/fully_connected.cc#L194
+  // LUCI_ASSERT(input_shape.rank() == 2 || input_shape.rank() == 3,
+  //             "Input rank of FullyConnected should be 2 or 3");
+
+  // https://github.com/tensorflow/tensorflow/blob/ea33c1e7a25d8025e8ee405ad8ab7be261798d76/tensorflow/lite/kernels/fully_connected.cc#L225
+  LUCI_ASSERT(weights_shape.rank() == 2, "Weights of FullyConnected should be 2");
```

Comment:
Negative test for this case, like give Weight with rank 1 or rank 3, looks must have.

Reply:
I add Neg_case to check weight with rank 1.
and also I add code and another NEG_case to check whether weights_shape is known or not.
If you find any issues, please feel free to let me know at any time. :)

Reply:
@Hanjin-Choi Sorry for late replies and your confusion 😢 

I had talk with @seanshpark and @shs-park. 
We have reached a consensus not to add negative tests for `LUCI_ASSERT`.
  * For consistency, `LUCI_ASSERT` in another files does not check in negative test. 
  * Since `assert` works differently in `release` and `debug` build, it is a bit troublesome to write the code for `assert`. 
You could remove assert related NEG tests. 

----------------------------------------------------

BTW, Since we should have more negative tests than positive tests (by SE policy), we need to find another place to add Neg tests. Adding another NEG tests can be work another PRs, not in this PR. -> Let's talk about this in other issues. 

/cc @shs-park

Reply:
As I followed your comments, I removed NEG_TCs to catch the Assertion.
If there are any problems, please let me know. Thanks.